tests/unit/gapic/vision_v1/test_product_search.py

```diff
@@ -291,6 +300,25 @@ def test_product_search_client_client_options(
             quota_project_id="octopus",
             client_info=transports.base.DEFAULT_CLIENT_INFO,
             always_use_jwt_access=True,
+            api_audience=None,
+        )
+    # Check the case api_endpoint is provided
+    options = client_options.ClientOptions(
+        api_audience="https://language.googleapis.com"
```

Comment:
What is an api_audience anyways? This is the vision client library, but here it is set to language? is this right?

Reply:
Spoke with @parthea offline; the use of "language" here this is an example api_audience; see this comment https://github.com/googleapis/python-api-core/pull/379/files#r921556538

Reply:
I'm going to resolve this thread since the issue is in the code generator repository (https://github.com/googleapis/gapic-generator-python/issues/1351).